.circleci/run-firesim-tests.sh

```diff
@@ -12,10 +12,19 @@ export FIRESIM_ENV_SOURCED=1
 
 SIMULATION_ARGS="${mapping[$1]}"
 
+cd $LOCAL_CHIPYARD_DIR/sims/firesim
```

Comment:
Why is this repeated?

Reply:
Because the libraries these generate are not cached in do-firesim-build.

I suppose the more correct solution would be to build these in the `build_tools` step, but that requires cloning all the firesim stuff in build_tools, which I judged to be more undesirable.

Blame @sagark for not trusting packaged libraries.